docs/input.md

```diff
@@ -229,21 +244,40 @@ configurations of the fluid's concentrations:
 ```
 input:
   ...
-  n: [[0.1, 0.3, 0.6], [0, 0.4, 0.4], [0.2, 0.8, 0], [0.5, 0.25, 0.25]]
+  current_gas_mix_ratios: [[0.1, 0.3, 0.6], [0, 0.4, 0.4], [0.2, 0.8, 0], [0.5, 0.25, 0.25]]
   ...
 ```
 
 We have used two sets of braces (a "list of lists") to indicate that the
-parameter `n` assumes 4 values, each of which is a list of 3 numbers.
+parameter `current_gas_mix_ratios` assumes 4 values, each of which is a list of 3 numbers.
 
 This is a very powerful syntax, but it comes with ѕome caveats:
 
-* You can't use the uniform or logarithmic spacing options with array parameters
+* You can't use logarithmic spacing options with array parameters
 * Skywalker makes no attempt to verify that all values in a list of array
   parameters have the same length
 * Single values for array parameters must be surrounded by two sets of braces
-  (e.g. `n: [[0.1, 0.3, 0.5]]`). Otherwise, Skywalker interprets the parameter
+  (e.g. `current_gas_mix_ratios: [[0.1, 0.3, 0.5]]`). Otherwise, Skywalker interprets the parameter
   as a list of multiple scalar parameters.
 
+```
+input:
+  ...
+  wet_geo_mean_diameter: [[0.001, 0.002, 0.003],[0.004, 0.005, 0.009], [0.001, 0.001, 0.002]]
+  ...
+```
+When using lattice input, This reads that the `wet_geo_mean_diameter` parameter assumes 
```

Comment:
Consider getting rid of  "This reads that" here. Otherwise, just lower-case the T.